src/onboarding2/Deployment/Deployment.js

```diff
@@ -171,25 +172,62 @@ function BoxReady({ opacity, boxTransform }) {
   )
 }
 
-const Deployment = React.memo(function Deployment({ ready }) {
+const Deployment = React.memo(function Deployment({
+  ready,
+  transactionsStatus,
```

Comment:
I think this is an array of all the transactions' statuses? Maybe it'd make sense to rename this to `transactionStatuses` instead (to make it clearer against a single status of all the transactions)?

Reply:
Yes :+1: